router/peers.go

```diff
@@ -25,6 +25,13 @@ type NameCollisionError struct {
 
 type PeerNameSet map[PeerName]struct{}
 
+type ConnectionView struct {
```

Comment:
Fowler calls this kind of thing a "Data Transfer Object"; if we adopted that pattern then `ConnectionDTO`.  `ConnectionData` not too bad but "Data" is not really adding much to anyone's understanding.